src/System.Runtime.Serialization.Primitives/ref/System.Runtime.Serialization.Primitives.cs

```diff
@@ -114,7 +114,27 @@ public partial class SerializationException : System.Exception
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public partial struct StreamingContext
     {
-        public override bool Equals(object obj) { return default(bool); }
-        public override int GetHashCode() { return default(int); }
+        public StreamingContext(StreamingContextStates state) { }
+        public StreamingContext(StreamingContextStates state, Object additional) { }
+       
+        public Object Context { get { throw null; } }       
+        public StreamingContextStates State  { get { throw null; } }
+
+        public override bool Equals(object obj) { throw null; }
+        public override int GetHashCode() { throw null; }
+    }
+    
+    [Flags]
+    public enum StreamingContextStates
+    {
+        //CrossProcess = 0x01,
```

Comment:
Why are you commenting out all the enum values? What is wrong with just providing them?